finality-aleph/src/testing/mocks/validator_network.rs

```diff
@@ -160,33 +162,44 @@ impl AsyncRead for UnreliableDuplexStream {
 pub struct UnreliableSplittable {
     incoming_data: UnreliableDuplexStream,
     outgoing_data: UnreliableDuplexStream,
+    peer_address: Address,
 }
 
 impl UnreliableSplittable {
     /// Create a pair of mock splittables connected to each other.
-    pub fn new(max_buf_size: usize, ends_after: Option<usize>) -> (Self, Self) {
+    pub fn new(
+        max_buf_size: usize,
+        ends_after: Option<usize>,
+        peer_address: Address,
```

Comment:
Shouldn't this take a pair of `Address`es?

Reply:
shouldy's